main.nf

```diff
@@ -804,15 +813,15 @@ process stringtieFPKM {
     stdout into stringtie_log
 
     script:
-    def StringTie_direction = ''
+    def st_direction = ''
```

Comment:
This one was a minor stylistic thing. Class names typically begin with a Capital, variables usually lower case. I wouldn't have noticed it normally, but my syntax highlighting showed it up.